Cargo.toml

```diff
@@ -44,7 +44,7 @@ features = ["derive"]
 version = "^1.0"
 
 [dependencies.apache-avro]
-version = "^0.14"
+version = "^0.16"
```

Comment:
I pushed it an hour ago. It is not yet announced! :-)

Reply:
I was running  `cargo-outdated` on a project to update some dependencies and happened to catch this. No idea it was that fresh :joy: